dashboard/src/components/InfoCard/Card/CardText/CardText.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020-2023 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from "react";
+
+export interface ICardTextProps {
+  children: React.ReactNode;
+}
+
+const CardText = ({ children }: ICardTextProps) => {
+  return <div className="card-text">{children}</div>;
+};
+
+export default CardText;
```

Comment:
Same here. Is it a react thing to break the elements up like this? Seems like a `/InfoCard/Card` could just have the text and image without splitting those out into separate components?

Reply:
100% agree, no actual benefits, especially if we ever plan to use the card from clarity at some point in the future. I thought the same while moving the code to tsx. I'll make a follow-up PR with the changes shortly.